packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/choice/accessories-choice.controller.js

```diff
@@ -115,6 +116,15 @@ export default class TelecomTelephonyLinePhoneAccessoriesChoiceCtrl {
 
   validateStep() {
     this.process.currentView = 'shipping';
+    this.atInternet.trackPage({
```

Comment:
Is the `trackPage` needed here ? As there is a hook on `ui router` state change , wan't the page be tracked twice ?

Reply:
Its a tab view so the page was not getting tracked